src/netspresso_trainer/dataloaders/classification/transforms.py

```diff
@@ -1,21 +1,29 @@
+import inspect
 from typing import Optional
 
 from torchvision.transforms.functional import InterpolationMode
 
 from ..augmentation import custom as TC
+from ..augmentation.registry import TRANSFORM_DICT
 from ..utils.constants import IMAGENET_DEFAULT_MEAN, IMAGENET_DEFAULT_STD
 
 
 def transforms_custom_train(conf_augmentation):
     assert conf_augmentation.img_size > 32
-    primary_tfl = [TC.RandomResizedCrop(conf_augmentation.img_size, interpolation=InterpolationMode.BILINEAR),
-                   TC.RandomHorizontalFlip(p=conf_augmentation.fliplr)
-                   ]
-    preprocess = [
+    preprocess = []
+    for augment in conf_augmentation.augment_info:
+        name = augment.name.lower()
+        transform_args = list(inspect.signature(TRANSFORM_DICT[name]).parameters)
+        transform_kwargs = {key:augment[key] for key in transform_args if hasattr(augment, key)}
```

Comment:
I personally believe that it is better to raise an error if there is no matching variable name, rather than ignoring the yaml config.
This is the current policy for yaml customization in overall codes.

Reply:
Oh, I got it. :+1: This coding style is quite different with current policy.
I will change it to produce error when config and arguments names are not matched. 
Then, also `inspect` will be removed.

Reply:
@illian01 

Thank you for consideration.
Let's discuss in later whether to notice/accept the yaml config with wrong format.